lib/teleterm/daemon/daemon_test.go

```diff
@@ -505,7 +506,7 @@ func TestImportantModalSemaphore(t *testing.T) {
 		t.Error("relogin completed successfully without acquiring the important modal semaphore")
 	case <-sphaErrC:
 		t.Error("sendPendingHeadlessAuthentication completed successfully without acquiring the important modal semaphore")
-	case <-time.After(5 * customWaitDuration):
+	case <-time.After(10 * customWaitDuration):
```

Comment:
Maybe bump `customWaitDuration` to 100ms or 200ms? It's still pretty fast, but not as punishing. (We could drop the `2*` then.)

Reply:
Having a short wait duration increases the speed of the test overall, so I think we are better off as is. I'll change this to just be 100ms as there isn't really a connection to the wait duration here.